packages/terra-table/src/Table.jsx

```diff
@@ -187,6 +182,13 @@ function Table(props) {
   // -------------------------------------
   // useEffect Hooks
 
+  useEffect(() => {
+    if (pinnedColumns.length === 0) {
+      // eslint-disable-next-line no-console
+      console.warn(ERRORS.PINNED_COLUMNS_UNDEFINED);
+    }
+  }, [pinnedColumns.length]);
```

Comment:
We would not want the update on the length of the columns.  We just need it to update when pinnedColumns does.

Reply:
Removed this and the sending of the cell coordinates.